### ISSUE TITLE
Formatting issue in the editor

### ISSUE THREAD

**Issue by LK-ML** [freescout-help-desk/freescout#4354]
Hi there,

first of all: thank you for Freescout! :)
We face an issue with the official saved response module. 
When writing a reply, then using a saved response, it sometimes will be enclosed into a  " <span style="font-size: 13.4px;">" .
This leads to the formatting being off. 

I have previously removed all styles from the saved responses, however the span is still being added sometimes. I am trying to replicate the issue.

Anwendungs-Version | 1.8.157
PHP 8.3.6
Are you using CloudFlare: No
Are you using non-official modules: No

**Comment by freescout-help** [comment]
Can you share some screenshots and HTML of the saved reply?
Can you reproduce the issue on demo.freescout.net?

**Comment by LK-ML** [comment]
Yes, the issue can be reproduced in demo.freescout.net:

1) I removed the <span> tag from the saved reply "General Answer" in the demo. I saved that and confirmed the new version has no span. 

![no span](https://github.com/user-attachments/assets/8ef7abdf-0ddd-4ccb-b337-7db29f90cf71)

2) Then I answered an existing ticket, wrote a few words, then used the Saved Reply "General Answer" The saved reply is added correctly without the span at first.

![no span in ticket](https://github.com/user-attachments/assets/844c1e5a-91ba-4df3-ad8e-4be38fa576a6)
![no span](https://github.com/user-attachments/assets/fe396506-0ab2-4c05-b313-28cfabc5986a)

3)
Removing the line break before the Saved Reply text moves the Saved Reply to the end of the previous text. 

![hi-edited](https://github.com/user-attachments/assets/7fcfe9f0-437d-461d-945a-6b9202f5a429)

The span-tag is added automatically after this removal:

![after editing](https://github.com/user-attachments/assets/15f79ff5-85a9-4570-a999-8b7b4a9b7cd2)

**Comment by freescout-help** [comment]
It's still difficult to figure out what exactly you are doing.

**Comment by LK-ML** [comment]
Sorry for that - let me try again :).

To replicate the issue in the demo:

1) Create a new ticket. Write "hello world". 

2) Use a Saved Reply, which has no styles applied - like "General Answer". Its applied correctly. Check in the code view - no span-tag.

3) Click before the first letter of the saved reply. In the Demo, this is before H of "Hi". Press backspace to delete the break before the H. Open code view - the "Hi" gets enclosed in a span with a font size 13.4px.

4) Open Saved Replies. Edit "General Answer". Open Code View - no span-Tag in the Saved Reply. 

This is not an issue in demo. When using saved replies to have common answers, the saved reply will end up with font size 13.4px, while the rest is a different font size. The mail looks bad :). 

Is this a bit more helpful?

**Comment by freescout-help** [comment]
The text looks fine even with the span tag added:

![1 (2)](https://github.com/user-attachments/assets/e3ccdce4-cd83-4e2e-b412-bd7130a188c7)

The code:

![1 (1)](https://github.com/user-attachments/assets/abaefa49-aa39-4738-8614-6907dce9b39a)

**Comment by LK-ML** [comment]
Agreed. 

When I do the same in my installation and send the mail out, the font size is shown in the coonversation differently. Part big, part small. Same when checking the original html mail. 

However, when I forward this conversation to me, the font size seems fine aswell. So, I can not say if outgoing mail is affected.

![Screenshot 2024-11-17 152647](https://github.com/user-attachments/assets/6c2d3deb-477f-4787-ab17-9f95303e7288)

**Comment by freescout-help** [comment]
Fixed in the `master` branch and will be published in the next release.

**Comment by LK-ML** [comment]
Thank you!

**Comment by kluvi** [comment]
Unfortunately it looks, that this fix breaks formatting in editor...
As you can see on screenshot, the text in editor is not bold.

![image](https://github.com/user-attachments/assets/a7e09ccc-d47f-4b82-b190-5283a6d10893)
![image](https://github.com/user-attachments/assets/62449167-f921-476f-958a-437067fb0794)
